cg/cli/upload/clinical_delivery.py

```diff
@@ -56,7 +56,7 @@ def upload_clinical_delivery(context: click.Context, case_id: str, dry_run: bool
 
     rsync_api: RsyncAPI = RsyncAPI(context.obj)
     is_complete_delivery, job_id = rsync_api.slurm_rsync_single_case(
-        case_id=case_id,
+        case=case_obj,
```

Comment:
```suggestion
        case=case,
```

in others parts of the code we try to refrain from using siffuxies like _obj but keep it simple. case for singular and cases for plural if it is about the model object